### PR TITLE
VD-42 removing external links for mapveu

### PR DESCRIPTION
**Pull request by moontrip**
this is in part related to:
https://redmine.apidb.org/issues/39536

### CODE REVIEW

**Comment by moontrip** [comment]
Tests can be made at:
https://dkwon.vectorbase.org/popbio-map/web/

File: web/js/map.Legend.js [hunk 1]
```diff
@@ -484,14 +484,20 @@ L.Control.MapLegend = L.Control.extend({
...
+            //     ' resistance/susceptibility. ' +
+            //     '<span class="active-others" data-toggle="modal" data-target="#ir-normalisation-help">' +
+            //     'More info</span></p>';
             inHtml += '</div></div>' +
                 '<div class="max-value" style="border: 0;">High</div></div>' +
                 '<p>' +
                 'Values have been rescaled globally and only give a relative indication of' +
                 ' resistance/susceptibility. ' +
-                '<span class="active-others" data-toggle="modal" data-target="#ir-normalisation-help">' +
-                'More info</span></p>';
```
Comment (bobular) [inline web/js/map.Legend.js]: Hi DK,
Sorry, I didn't mean for the "More info" link to be removed.
Could it be put back (and perhaps fix the spacing) please?

Then fix the broken image URLs (copy the images into the repo).

many thanks,
Bob

Comment (moontrip) [reply web/js/map.Legend.js]: Ah, I got it! Will work for that probably tomorrow as I have meetings

**Comment by moontrip** [comment]
implemented Bob's suggestion

**Pull request merged**